docs/external_integrations/authorization_ory_kratos/handling_webhooks.md

```diff
@@ -0,0 +1,183 @@
+# Handling webhooks
+
+To effectively manage incoming webhooks from Ory Kratos, you can employ the [KratosWebHookHandlerBase] class. The following example demonstrates how to synchronize identity data received from Kratos through webhooks with your database using [MassTransit]. This assumes that your Kratos instance is properly configured to send webhooks to your application.
+
+## Packages
+
+| Package | Link | Application in section |
+| --- | ----------- | ----------- |
+| LeanCode.Kratos | [![NuGet version (LeanCode.Kratos)](https://img.shields.io/nuget/vpre/LeanCode.Kratos.svg?style=flat-square&logo=nuget)](https://www.nuget.org/packages/LeanCode.Kratos/8.0.2260-preview/) | Webhook handling |
+
+## Mapping the Endpoint in Application Configuration
+
+First, you should map the `KratosIdentitySyncHandler` class to the `/kratos/sync-identity` endpoint in your application's configuration:
+
+```csharp
+protected override void ConfigureApp(IApplicationBuilder app)
+{
+    // . . .
+
+    app.UseAuthentication()
+        .UseEndpoints(endpoints =>
+        {
+            // . . .
+
+            endpoints.MapPost(
+                "/kratos/sync-identity",
+                ctx => ctx.RequestServices
+                    .GetRequiredService<KratosIdentitySyncHandler>()
+                    .HandleAsync(ctx)
+            );
+
+            // . . .
+        });
+
+    // . . .
+}
+```
+
+## Defining the Webhook Handler
+
+Then, you can define the webhook handler to handle incoming requests. In this example, we deserialize request into [Identity] class, verify it inside a webhook and send an event that the identity has been updated, which will be handled by the appropriate `IConsumer` using [MassTransit]:
+
+```csharp
+public partial class KratosIdentitySyncHandler : KratosWebHookHandlerBase
+{
+    private readonly IBus bus;
+
+    public KratosIdentitySyncHandler(
+        KratosWebHookHandlerConfig config,
+        IBus bus)
+        : base(config)
+    {
+        this.bus = bus;
+    }
+
+    protected override async Task HandleCoreAsync(HttpContext ctx)
+    {
```

Comment:
Might be worth mentioning that at this point we have already handled API key check in the base class so it's not required to do this here again.